src/sentry/workflow_engine/handlers/condition/event_frequency_handlers.py

```diff
@@ -0,0 +1,245 @@
+import contextlib
+from abc import ABC, abstractmethod
+from collections import defaultdict
+from collections.abc import Callable, Mapping
+from datetime import datetime, timedelta
+from typing import Any, Literal, TypedDict
+
+from django.db.models import QuerySet
+
+from sentry import tsdb
+from sentry.issues.constants import get_issue_tsdb_group_model
+from sentry.issues.grouptype import GroupCategory, get_group_type_by_type_id
+from sentry.models.group import Group
+from sentry.tsdb.base import TSDBModel
+from sentry.utils import json
+from sentry.utils.iterators import chunked
+from sentry.utils.snuba import options_override
+from sentry.workflow_engine.models.data_condition import Condition
+from sentry.workflow_engine.registry import condition_handler_registry
+from sentry.workflow_engine.types import DataConditionHandler, DataConditionResult
+
+SNUBA_LIMIT = 10000
+STANDARD_INTERVALS: dict[str, tuple[str, timedelta]] = {
```

Comment:
is all of this copied instead of imported because we'll be removing the existing code? it might be useful to leave comments in the original code pointing to this and saying it needs to stay in sync in case another team changes something

Reply:
hmm i could just import this instead and change it later